### ISSUE TITLE
About the augment data

### ISSUE THREAD

**Issue by haha010508** [phonexiaresearch/VBx-training-recipe#3]
I have a question, why the augment data is so big, isn't it slow to train?Have you considered limiting the augment data  size? like the voxceleb v2 done? thanks!

**Comment by haha010508** [comment]
And i got this error:

local/nnet3/xvector/get_egs_but.sh: Preparing train and validation lists
local/nnet3/xvector/get_egs_but.sh: Producing archives for training. Number of archives will estimated based on the training data.
local/nnet3/xvector/get_egs_but.sh: Allocating training examples
local/nnet3/xvector/get_egs_but.sh: Allocating training subset examples
local/nnet3/xvector/get_egs_but.sh: Generating training examples on disk
local/nnet3/xvector/get_egs_but.sh: Generating training subset examples on disk
local/nnet3/xvector/get_egs_but.sh: Shuffling order of archives on disk
rm: cannot remove 'exp/egs/valid_egs_temp.1.ark': No such file or directory
cat: exp/egs/valid_egs.1.scp: No such file or directory

**Comment by Jamiroquai88** [comment]
Hi, it is indeed slow. It was trained in parallel on 6 GPUs. You are missing `exp/egs/valid_egs_temp.1.ark` but you probably don't need that.

**Comment by Jamiroquai88** [comment]
@MichalKlco

**Comment by haha010508** [comment]
ok, thanks for your reply, i will try it later, but  i do not understand if no valid data, how to valid the train result?  i found some code about the valid_egs_temp.1.ark was commented, need i to uncomment it? the code in VBx-training-recipe-main/local/nnet3/xvector/get_egs_but.sh

**Comment by Jamiroquai88** [comment]
Hi @haha010508, sorry for the late reply, we are still kind of juggling the responsibilities for this repository. No validation set is used in the recipe, feel free to PR if you believe this is an important feature. Kaldi had some issues with creating validation sets, so I just commented it out since it is not used.
